src/trace_reconstruction.py

```diff
@@ -6,54 +6,70 @@
 
 
 def process_arguments():
-    global image_path, csv_path
+
+    global image_path, csv_path, reconstruction_timestamp
     parser = argparse.ArgumentParser(description="Script to reconstruct data from a CSV file ")
     parser.add_argument("--image-path", required=True, help="Path to image file")
-    parser.add_argument("--csv-path", required=True, help="Path to the csv file")
+    parser.add_argument("--csv-path", required=True, help="Path to folder containing CSV files")
+    parser.add_argument("--date", required=True, help="date in dd.mm.yyyy")
+    parser.add_argument("--time", required=True, help="time in hh:mm:ss")
     args = parser.parse_args()
     image_path = args.image_path
     csv_path = args.csv_path
+    date_string = args.date
+    time_string = args.time
+
+    date_obj = datetime.datetime.strptime(date_string, "%d.%m.%Y")
+    time_obj = datetime.datetime.strptime(time_string, "%H:%M:%S")
+    datetime_reconstruction_timestamp = datetime.datetime.combine(date_obj.date(), time_obj.time())
+
+    reconstruction_timestamp = int(datetime_reconstruction_timestamp.timestamp()) * 1000
+    #YYYY-MM-DDThh_mm_ssSmmm
 
-date_string = "18.05.2023"
-time_string = "14:22:02"
+def convertToDateTime(csv_list):
```

Comment:
Convert function name to `snake _case` to be synchronized with all the others

Reply:
resolved in [41036d9](https://github.com/INFB-Projekt/flashspeicher-rekonstruktion/pull/22/commits/41036d9f990fbee8a86b498aa857c4556409fb70)